index.html

```diff
@@ -38,8 +38,16 @@
             </ul>
           
         </nav>
+        
       </header>
-
+      <section class="menuBar">
+        <div class="closeMenu">X</div>
+        <ul>
+          <li><a class="menuLinks"  href="#portfolio">Portfolio</a></li>
+          <li><a class="menuLinks" href="#about">About</a></li>
+          <li><a class="menuLinks" href="#contact">Contact</a></li>
```

Comment:
- [ ] You have done incredibly well to implement the mobile menu functionality.  To improve on the design, I suggest considering aligning the links a little bit to the left and also adding spacing on top of the link text. This is to ensure that as the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) suggest, you try as much as possible to ensure that The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins) ☺️.